langwatch/prisma/migrations/20250803091127_drop_prompt_version_seq/migration.sql

```diff
@@ -0,0 +1,6 @@
+-- Automatic sequence actually doesn't make sense for version table, as the sequence is not global but per prompt.
+ALTER TABLE "LlmPromptConfigVersion" ALTER COLUMN "version" DROP DEFAULT;
+DO $$
+BEGIN
+    DROP SEQUENCE IF EXISTS "LlmPromptConfigVersion_version_seq";
+END $$;
```

Comment:
I also figure out this one, it was back from a migration that was applied to dev db for some reason but never commited